Tutorials/intro-to-octave/octave_introduction.py

```diff
@@ -319,59 +335,61 @@
 ###########################
 external_clock = False
 if external_clock == "10MHz":
-    qmm.octave_manager.set_clock("octave1", ClockType.External, ClockFrequency.MHZ_10)
+    qmm.octave_manager.set_clock(octave, ClockType.External, ClockFrequency.MHZ_10)
```

Comment:
It is written in Pycharm "Do not use OctaveManager, it will be removed in the next version", it is normal right?

Reply:
I think it's fine. @yuvalqm ?

Reply:
yes, It is going for depracation